kustomize/azure/provider.yaml

```diff
@@ -2,8 +2,10 @@ apiVersion: azure.crossplane.io/v1alpha3
 kind: Provider
 metadata:
   name: azure-provider
+  annotations:
+    templatestacks.crossplane.io/deletion-priority: "-1"
 spec:
   credentialsSecretRef:
-    namespace: $(CRED_SECRET_NAMESPACE)
-    name: $(CRED_SECRET_NAME)
-    key: $(CRED_SECRET_KEY)
\ No newline at end of file
+    name: azure-account-creds
```

Comment:
Is this change just because these values get overwritten?

Reply:
Yes, we must have forgotten to change them. They are overwritten anyway but these values are better defaults.